`poetry install` failed with error about gmpy2

### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

recently

### Cloud or Self Hosted

Self Hosted (Source)

### Steps to reproduce

1. start project in vscode with devcontainer
2. the devcontainer building process will end up with a error message like this:
```
Package operations: 1 install, 0 updates, 0 removals

  - Installing gmpy2 (2.2.1): Failed

  ChefBuildError

  Backend subprocess exited when trying to invoke build_wheel
  
  running bdist_wheel
  running build
  running build_py
  creating build/lib.linux-aarch64-cpython-310/gmpy2
  copying gmpy2/__init__.py -> build/lib.linux-aarch64-cpython-310/gmpy2
  running egg_info
  writing gmpy2.egg-info/PKG-INFO
  writing dependency_links to gmpy2.egg-info/dependency_links.txt
  writing requirements to gmpy2.egg-info/requires.txt
  writing top-level names to gmpy2.egg-info/top_level.txt
  reading manifest file 'gmpy2.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  adding license file 'COPYING'
  adding license file 'COPYING.LESSER'
  writing manifest file 'gmpy2.egg-info/SOURCES.txt'
  copying gmpy2/__init__.pxd -> build/lib.linux-aarch64-cpython-310/gmpy2
  copying gmpy2/gmpy2.h -> build/lib.linux-aarch64-cpython-310/gmpy2
  copying gmpy2/gmpy2.pxd -> build/lib.linux-aarch64-cpython-310/gmpy2
  running build_ext
  building 'gmpy2.gmpy2' extension
  creating build/temp.linux-aarch64-cpython-310/src
  gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/tmp/tmpf3uh8tfo/.venv/include -I/usr/local/include/python3.10 -c src/gmpy2.c -o build/temp.linux-aarch64-cpython-310/src/gmpy2.o -DSHARED=1
  In file included from src/gmpy2.c:112:
  src/gmpy2.h:77:10: fatal error: mpfr.h: No such file or directory
     77 | #include <mpfr.h>
        |          ^~~~~~~~
  compilation terminated.
  error: command '/usr/bin/gcc' failed with exit code 1
  

  at /usr/local/py-utils/venvs/poetry/lib/python3.10/site-packages/poetry/installation/chef.py:164 in _prepare
      160│ 
      161│                 error = ChefBuildError("\n\n".join(message_parts))
      162│ 
      163│             if error is not None:
    → 164│                 raise error from None
      165│ 
      166│             return path
      167│ 
      168│     def _prepare_sdist(self, archive: Path, destination: Path | None = None) -> Path:

Note: This error originates from the build backend, and is likely not a problem with poetry but with gmpy2 (2.2.1) not supporting PEP 517 builds. You can verify this by running 'pip wheel --no-cache-dir --use-pep517 "gmpy2 (==2.2.1)"'.
```
If you run `poetry install` in terminal. You will reproduce same error.

### ✔️ Expected Behavior

`poetry install` should run successfully.

### ❌ Actual Behavior

Install error about `gmpy2`.

--- a/.devcontainer/post_start_command.sh
+++ b/.devcontainer/post_start_command.sh
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-poetry install -C api
+cd api && poetry install

